code/modules/cargo/supplypod.dm

```diff
@@ -200,7 +199,7 @@
 /obj/structure/closet/supplypod/Destroy()
 	if (!opened) //If we havent opened yet, we're opening because we've been destroyed. Lets dump our contents by opening up
 		open(src, broken = TRUE)
-	return ..()
+	. = ..()

```

Comment:
lol why did you change this

Reply:
Force of habit, sorry. At some point Remie told me that . was faster than using return.